### ISSUE TITLE
Device mapping interface improvements (Trac #6)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#10]
Setting the channel number for each electrode on a large device is currently pretty tedious.  A couple of ways to make this process easier are:

 1. Autoincrement the channel number after each electrode assignment.
 2. Give the OK button focus so that the user can just press enter after typing the channel number.

Migrated from http://microfluidics.utoronto.ca/ticket/6
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Setting the channel number for each electrode on a large device is currently pretty tedious.  A couple of ways to make this process easier are:\n\n 1. Autoincrement the channel number after each electrode assignment.\n 2. Give the OK button focus so that the user can just press enter after typing the channel number.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "wontfix", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Device mapping interface improvements", 
    "priority": "minor", 
    "keywords": "interface", 
    "version": "0.1", 
    "time": "2011-09-22T01:26:27", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20110922T01:28:49: ryan changed description from:

> Setting the channel number for each electrode on a large device is currently pretty tedious.  A couple of ways to make this process easier are:
> 
>  1. Autoincrement the channel number after each electrode assignment.
>  1. Give the OK button focus so that the user can just press enter after typing the channel number.

to:

> Setting the channel number for each electrode on a large device is currently pretty tedious.  A couple of ways to make this process easier are:
> 
>  1. Autoincrement the channel number after each electrode assignment.
>  2. Give the OK button focus so that the user can just press enter after typing the channel number.

**Comment by cfobel** [comment]
Trac update at 20120403T14:42:19: 
* ryan commented:

> Part 2. is implemented in Ticket #75. Part 1. is probably not really necessary.


* ryan changed resolution from "" to "wontfix"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "gui" to "microdrop"
